rust/src/common.rs

```diff
@@ -1,3 +1,5 @@
+//! Utility library module for handling strings, hexadecimals and bytes.
```

Comment:
As with a few other cases, it's probably better if we try to be a bit more generic here, to avoid this comment getting outdated too soon.

Suggestion:
`Utility library module for commonly used string, hex, and other elements.`

Reply:
OK!

Doing that.